api/lib/mno_enterprise/concerns/controllers/jpi/v1/current_users_controller.rb

```diff
@@ -54,6 +54,8 @@ def register_developer
   def update_password
     @user = current_user
     @user = @user.update_password(data: {attributes: password_params}).first
+    # TODO: [APIv2] This is not working!
+    # update_password returns an empty array in case of error
```

Comment:
Should this be addressed in a separate issue?

Reply:
Yes, it's part of [MNOE-602].
I left the comment there for grep'ing :)